[DOC] README indicates the lowest supported TypeScript version (#2251)
process-analytics/bpmn-visualization-js: README.md

#### 📜 TypeScript Support

> `bpmn-visualization` provides type definitions, so the integration works out of the box in TypeScript projects.
> `bpmn-visualization` requires TypeScript 4.5 or greater.

Author: ```suggestion
`bpmn-visualization` provides type definitions, so the integration works out of the box in TypeScript projects.

`bpmn-visualization` requires **TypeScript 4.5** or greater.
```

#### 📜 TypeScript Support

`bpmn-visualization` provides type definitions, so the integration works out of the box in TypeScript projects.

`bpmn-visualization` requires **TypeScript 4.5** or greater.

> Depending on the build system used by the TypeScript project, it may get the following type errors:

Author: ```suggestion
Depending on the TypeScript configuration of the project, it may get the following type errors:
```
